sql/core/src/main/scala/org/apache/spark/sql/UDFRegistration.scala

```diff
@@ -488,219 +488,241 @@ class UDFRegistration private[sql] (functionRegistry: FunctionRegistry) extends
    * @since 1.3.0
    */
   def register(name: String, f: UDF1[_, _], returnType: DataType): Unit = {
+    val func = f.asInstanceOf[UDF1[Any, Any]].call(_: Any)
```

Comment:
There is commented out code above thats used to generate these functions.  We should update it or delete it.